src/test/java/uk/gov/hmcts/divorce/solicitor/event/SolicitorConfirmServiceTest.java

```diff
@@ -93,6 +95,7 @@ public void shouldAddAnyConfirmServiceAttachmentsToDocumentsUploadedList() {
         final CaseData caseData = caseData();
         caseData.getApplication().setSolSignStatementOfTruth(YES);
         caseData.getApplication().setServiceMethod(SOLICITOR_SERVICE);
+        caseData.setApplicationType(ApplicationType.SOLE_APPLICATION);
```

Comment:
Can we add a test to confirm the state doesn't move when the Aos date is set?

Reply:
Already done, just may need relocating.

Reply:
Added to SetConfirmServiceStateTest after refactor.